hoodie-client/src/main/java/com/uber/hoodie/WriteStatus.java

```diff
@@ -31,11 +31,11 @@
  */
 public class WriteStatus implements Serializable {
 
-  private final HashMap<HoodieKey, Throwable> errors = new HashMap<>();
+  protected final HashMap<HoodieKey, Throwable> errors = new HashMap<>();
```

Comment:
Do you still need protected access ? These container attributes have getters with which you can update the container ?

Reply:
+1